===================================================================
RCS file: /home/kde/kdebindings/qtjava/javalib/docs/en/Makefile.am,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- qtjava/javalib/docs/en/Makefile.am	2004/08/27 17:20:17	1.4
+++ qtjava/javalib/docs/en/Makefile.am	2005/01/12 15:47:07	1.5
@@ -4,22 +4,22 @@
 EXTRA_DIST = index.html index-1.html index-2.html index-3.html index-4.html index-5.html index-6.html 
 
 install-data-local:
-	$(mkinstalldirs) $(DESTDIR)$(docdir)/javalib/
-	$(INSTALL_DATA) $(srcdir)/index.html $(DESTDIR)$(docdir)/javalib/index.html
-	$(INSTALL_DATA) $(srcdir)/index-1.html $(DESTDIR)$(docdir)/javalib/index-1.html
-	$(INSTALL_DATA) $(srcdir)/index-2.html $(DESTDIR)$(docdir)/javalib/index-2.html
-	$(INSTALL_DATA) $(srcdir)/index-3.html $(DESTDIR)$(docdir)/javalib/index-3.html
-	$(INSTALL_DATA) $(srcdir)/index-4.html $(DESTDIR)$(docdir)/javalib/index-4.html
-	$(INSTALL_DATA) $(srcdir)/index-5.html $(DESTDIR)$(docdir)/javalib/index-5.html
-	$(INSTALL_DATA) $(srcdir)/index-6.html $(DESTDIR)$(docdir)/javalib/index-6.html
+	$(mkinstalldirs) $(DESTDIR)$(kde_htmldir)/en/javalib/
+	$(INSTALL_DATA) $(srcdir)/index.html $(DESTDIR)$(kde_htmldir)/en/javalib/index.html
+	$(INSTALL_DATA) $(srcdir)/index-1.html $(DESTDIR)$(kde_htmldir)/en/javalib/index-1.html
+	$(INSTALL_DATA) $(srcdir)/index-2.html $(DESTDIR)$(kde_htmldir)/en/javalib/index-2.html
+	$(INSTALL_DATA) $(srcdir)/index-3.html $(DESTDIR)$(kde_htmldir)/en/javalib/index-3.html
+	$(INSTALL_DATA) $(srcdir)/index-4.html $(DESTDIR)$(kde_htmldir)/en/javalib/index-4.html
+	$(INSTALL_DATA) $(srcdir)/index-5.html $(DESTDIR)$(kde_htmldir)/en/javalib/index-5.html
+	$(INSTALL_DATA) $(srcdir)/index-6.html $(DESTDIR)$(kde_htmldir)/en/javalib/index-6.html
 
 uninstall-local:
-	-rm -f $(DESTDIR)$(docdir)/javalib/index.html
-	-rm -f $(DESTDIR)$(docdir)/javalib/index-1.html
-	-rm -f $(DESTDIR)$(docdir)/javalib/index-2.html
-	-rm -f $(DESTDIR)$(docdir)/javalib/index-3.html
-	-rm -f $(DESTDIR)$(docdir)/javalib/index-4.html
-	-rm -f $(DESTDIR)$(docdir)/javalib/index-5.html
-	-rm -f $(DESTDIR)$(docdir)/javalib/index-6.html
+	-rm -f $(DESTDIR)$(kde_htmldir)/en/javalib/index.html
+	-rm -f $(DESTDIR)$(kde_htmldir)/en/javalib/index-1.html
+	-rm -f $(DESTDIR)$(kde_htmldir)/en/javalib/index-2.html
+	-rm -f $(DESTDIR)$(kde_htmldir)/en/javalib/index-3.html
+	-rm -f $(DESTDIR)$(kde_htmldir)/en/javalib/index-4.html
+	-rm -f $(DESTDIR)$(kde_htmldir)/en/javalib/index-5.html
+	-rm -f $(DESTDIR)$(kde_htmldir)/en/javalib/index-6.html
 
 ####### kdevelop will overwrite this part!!! (end)############
